### ISSUE TITLE
[Comprehension] User doesn't have enough information to complete a task.

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#76953]
_This ticket was generated during the pilot process for the new VA.gov experience standards. Please tag `@platform-governance-team-members` on Slack if you need assistance interpreting this ticket._

## Product Information
**Team:** Decision Reviews Team
**Product:** Board Appeal (Notice of Disagreement) VA Form 10182
**Feature:** Request an Extension Feature (OMB Approved No. 2900-0674)

## Findings details
**VA.gov Experience Standard - issue:** User doesn't have enough information to complete a task.
**VA.gov Experience Standard - category:** Comprehension
**Launch-blocking:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

In the pages included in this review, there are two instances of helper text that might be missed by screen reader users. Neither of these seem critical but both *possibly* provide context that's important for completing the task. Up to you all to decide if it actually is, since you know your users and your product better than me!

 When a screen reader user is moving through a form flow like this, they typically don't have their screen reader announce everything on the page --- it's just too much and too repetitive to listen to. Instead, they enter "forms mode" for some screen readers, or otherwise just press [tab] to move between interactive elements.

So when you tab to a form input, the screen reader will announce the label, the contents of the input, and any text that's associated with the input. When extra helper text is included in the `fieldset` but not programmatically associated with any inputs, it won't be announced by screen readers.

The two instances of helper text:
- On [Request an extension](https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/extension-request): "If you request an extension (extra time) to file VA Form 10182 for good cause, you’ll need to tell us why you have good cause."
- On [Reason for extension](https://staging.va.gov/decision-reviews/board-appeal/request-board-appeal-form-10182/extension-reason): "Tell us why you have good cause for an extension."

Both of these are the only instances where the phrase "good cause" is used to describe the reason for the extension. If that particular phrasing is important for understanding what should be provided, that phrase is likely to get missed by screen reader users.

<details><summary>Link, screenshot or steps to recreate</summary>

**Request an extension**
```
<fieldset class="vads-u-margin-y--2 rjsf-object-field">
	<div>
		<fieldset class="vads-u-margin-y--2 rjsf-object-field">
			<legend class="schemaform-block-title schemaform-block-subtitle" id="root_view:requestExtensionInfo__title"></legend>
			<h3 class="vads-u-margin-top--0">Request an extension</h3>
			<p class="vads-u-margin-y--0">If you request an extension (extra time) to file VA Form 10182 for good cause, you’ll need to tell us why you have good cause.</p>
		</fieldset>
	</div>
	<div>
		<va-radio class="rjsf-web-component-field hydrated" enableanalytics="true" label="Are you requesting an extension?" name="root_requestingExtension" required="false" uswds="true" value="true" aria-invalid="false">
			...
		</va-radio>
	</div>
</fieldset>
```

Note that the `legend` element is empty here. I'm guessing that's an error that snuck by.

**Reason for extension**
```
<fieldset class="vads-u-margin-y--2 rjsf-object-field">
	<legend class="schemaform-block-title" id="root__title">
		<h3 class="vads-u-margin-y--0">Reason for extension</h3>
	</legend>
	<p class="vads-u-margin-y--0">Tell us why you have good cause for an extension.</p>
	<div>
		<va-textarea class="rjsf-web-component-field hydrated" enableanalytics="false" hint="2300 characters max." label="Reason for requesting an extension:" maxlength="2300" name="root_extensionReason" required="true" uswds="true" value="..." charcount="true"></va-textarea>
	</div>
</fieldset>
```

</details>

## Recommended action
If you determine these instances of helper text should be announced to screen reader users, then my recommendation would be one of the following:
- Update the labels on the questions to include the relevant information, and remove the helper text.
- Include the helper text in the `hint` prop on the VA web components for those questions.
- Put the helper text inside the `legend` and style to look like plain text.

If they're not important enough to announce to screen reader users, consider whether they're needed at all and can be edited out entirely.

## References

- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** Platform OCTO leadership asks that we document all issues we identify. If you believe that this issue is out of scope or not your team's responsibility, it is still your responsibility to resolve the issue. Please comment on Slack in your team channel tagging `@platform-governance-team-members`, providing an explanation and who you believe is responsible. Governance team will research and follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by Mottie** [comment]
Related ticket #76909 & PR [#28183](https://github.com/department-of-veterans-affairs/vets-website/pull/28183).

**Comment by saderagsdale** [comment]
@shiragoodman This one should be ready for review/closeout.

**Comment by shiragoodman** [comment]
@saderagsdale We don't require validation by the Governance team, so if you believe this this is resolved, and your PO agrees, you're free to close. however, if you'd _like_ Governance team to validate, let us know. FYSA @briandeconinck
